README.md

```diff
@@ -1,65 +1,73 @@
-README
-======
+<p align="center"><a href="https://symfony.com" target="_blank">
+    <img src="http://symfony.com/logos/symfony_black_02.svg">
```

Comment:
Nice catch! Other than that, the only link that doesn't use HTTPS is semver.org ... because they seem to have some issues.